packages/firebase/package.json

```diff
@@ -13,6 +13,14 @@
     "realtime",
     "storage"
   ],
+  "files": [
+    "**/dist/",
+    "**/package.json",
+    "/firebase*.js",
+    "/firebase*.map",
+    "/index.d.ts",
+    "/externs"
```

Comment:
Do these get resolved as relative paths?

Reply:
Relative to the root of the package I believe yes.